torch/_tensor_docs.py

```diff
@@ -3733,6 +3733,36 @@ def callable(a, b) -> number
 ``x.T`` is equivalent to ``x.permute(n-1, n-2, ..., 0)``.
 """)
 
+add_docstr_all('real',
+               r"""
+Returns a new tensor containing real values of the :attr:`self` tensor.
+The returned tensor shares the same data as :attr:`self`. For complex tensors,
```

Comment:
nit: can you use "memory" instead of "data" here to avoid any confusion with `.data`.
Same below.